apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -599,9 +599,18 @@ handle_request_('GetStatus', _Params, _Context) ->
     Syncing = aec_sync:is_syncing(),
     SyncProgress = aec_sync:sync_progress(),
     Listening = true, %% TODO
-    Protocols = maps:fold(fun(Vsn, Height, Acc) ->
+    Protocols =
+        maps:fold(fun(Vsn, Height, Acc) ->
                           [#{<<"version">> => Vsn, <<"effective_at_height">> => Height} | Acc]
-                 end, [], aec_hard_forks:protocols()),
+                  end, [], aec_hard_forks:protocols()),
+    Version = aec_blocks:version(TopKeyBlock),
+    Protocols2 =
+        case aeu_env:get_env(aecore, fork, undefined) of
+            #{version := Version, signalling_end_height := SigEndHeight} ->
```

Comment:
If the `Version` is the same as top block's `Version`, it means the new protocol was activated.